### DISCUSSION TITLE
getting error while publishing the vendor

### DISCUSSION THREAD

**Discussion by bhagat-abhishek** [coderflexx/laravisit#36] [Q&A]
php artisan vendor:publish --provider="Coderflex\\Laravisit\\LaravisitServiceProvider"

   INFO  No publishable resources for tag [].

-- When i run the vendor:publish 

i get the below error and i am not able to run the migrate command from installing it.

**Comment by ousid** [comment]
Hey @bhagat-abhishek 

Sorry for the late response.

That's odd, I just created a fresh Laravel project, with `laravisit` package installed, and I found no issue.

Could you give more details on your Laravel project versions, and which version you are using in this package?

Thanks!
